x-pack/plugins/reporting/common/errors/index.ts

```diff
@@ -8,6 +8,12 @@
 /* eslint-disable max-classes-per-file */
 import { i18n } from '@kbn/i18n';
 
+export interface ReportingError {
+  /**
+   * Return a message describing the error that is human friendly
+   */
+  humanFriendlyMessage?(): string;
```

Comment:
I am just curious why you decided not to go with the built-in `toString()` method? That seems like it can very well fit into our use case.

Reply:
Good question!

In most cases on the server-side we will be using the default `toString` behaviour. But when storing this as a message on the Reporting document (where it is seen by users) sometimes we want to have a special human-friendlier version of the stringified error. Does that make sense? Is there another way to achieve this? Perhaps `toString('humanFriendly')`?

Reply:
Thanks, @jloleysens. That makes total sense now 👍

No need to overcomplicate this. The current solution is clear enough.